CuratedContent/TuringWay.md

```diff
@@ -0,0 +1,34 @@
+# Reproducible and ethical science. 
```

Comment:
Is this the best title for this CC article?  Perhaps something like "The  Turning Way: Reproducible and Ethical Science"?

Reply:
The title "Reproducible and ethical science" would make me thing this is a deep article on reproducability and ethics and would site may sources on the topic.  I think that adding the name of the article "The Turing Way" would eliminate such confusion.